pkg/core/resources/model/resource.go

```diff
@@ -409,12 +409,16 @@ func IsReferenced(refMeta ResourceMeta, refName string, resourceMeta ResourceMet
 }
 
 func IsLocallyOriginated(mode config_core.CpMode, r Resource) bool {
+	return IsLocallyOriginatedByLabels(mode, r.GetMeta().GetLabels())
+}
+
+func IsLocallyOriginatedByLabels(mode config_core.CpMode, labels map[string]string) bool {
```

Comment:
I think we should keep only one of the methods `IsLocallyOriginated` or `IsLocallyOriginatedByLabels`. Having both just makes no sense in my opinion

Reply:
removed